app/src/main/assets/html/error/css/styles.css

```diff
@@ -0,0 +1,173 @@
+*,
+*:before,
+*:after {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+/* open-sans-regular - latin */
```

Comment:
per @bluemarvin's [suggestion](https://github.com/MozillaReality/FirefoxReality/pull/170#issuecomment-405312861), feel free to replace _Open Sans_ with this sans-serif System font family:

```css
  font-family: BlinkMacSystemFont, -apple-system, Segoe UI, Roboto, Oxygen, Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
```